pkg/cloud/services/loadbalancer/loadbalancer.go

```diff
@@ -356,6 +363,12 @@ func (s *Service) getOrCreateMonitor(openStackCluster *infrav1.OpenStackCluster,
 		Timeout:        5,
 	}
 	monitor, err = s.loadbalancerClient.CreateMonitor(monitorCreateOpts)
+	// Skip creating monitor if it is not supported by Octavia provider
+	if strings.Contains(err.Error(), "501") {
+		record.Warnf(openStackCluster, "SkippedCreateMonitor", "Health Monitor is not implemented with the current provider")
```

Comment:
We should probably add an even explaining this too.

Reply:
Added event.